api/v1alpha1/stresschaos_types.go

```diff
@@ -120,6 +120,7 @@ func (in *Stressors) Normalize() (string, string, error) {
 		}
 	}
 	if in.CPUStressor != nil && in.CPUStressor.Workers != 0 {
+		cpuStressors += " --cpu-load-slice 10 --cpu-method sqrt "
```

Comment:
```suggestion
		cpuStressors += " --cpu-load-slice 10 --cpu-method sqrt"
```

the last whitespace seems not required, please remove it.